artifactory_cleanup/cli.py

```diff
@@ -112,9 +167,7 @@ def main(self):
         if self._policy:
             cleanup.only(self._policy)
 
-        table = PrettyTable()
-        table.field_names = ["Cleanup Policy", "Files count", "Size"]
-        table.align["Cleanup Policy"] = "l"
+        result = {"policies": [], "total_size": None}
```

Comment:
`total_size` must be 0 by default - if there's no policies it must show actual size